src/drop-flags.md

```diff
@@ -79,5 +78,8 @@ if condition {
 }
 ```
 
-The drop flags are tracked on the stack and no longer stashed in types that
-implement drop.
+At Rust 1.0, these flags were stored in the actual values that needed to
+be tracked. This was a big mess and you had to worry about it in unsafe code.
+
+As of Rust 1.13, these flags are stored seperately on the stack, so you no
```

Comment:
"Starting from", rather than "As of". The flags will not go back into actual values in a way you have to care about.